pkg/component/generic/collection/v0/config/tasks.json

```
@@ -316,5 +316,62 @@
       "title": "Output",
       "type": "object"
     }
+  },
+  "TASK_CONCAT": {
+    "instillShortDescription": "Concat the arrays. i.e. `[1, 2] + [3, 4] = [1, 2, 3, 4]`",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "arrays"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "arrays": {
+          "description": "Specify the arrays you want to concat.",
```

nit: this should be a description, not an order
```suggestion
          "description": "The arrays to be concatenated.",
```

---

🗣️ not crucial for this PR, just opening a discussion

We need to decide about the periods for the task tables. 2 options:
- Always finish with a period
- Don't finish with a period by default, only do so when we have > 1 sentence in the description (separated by a period).

cc @GeorgeWilliamStrong

---

> Always finish with a period

Now, I follow this, which @GeorgeWilliamStrong mentioned it somewhere.